rebar.config.erl

```diff
@@ -457,6 +470,22 @@ list_dir(Dir) ->
     {ok, Names} = file:list_dir(Dir),
     [list_to_atom(Name) || Name <- Names, filelib:is_dir(filename:join([Dir, Name]))].
 
+get_compile_secret(SecretFile) ->
+    {ok, Secret} = file:read_file(SecretFile),
+    string:strip(binary_to_list(Secret), right).
```

Comment:
this strips only white spaces.

```suggestion
    string:trim(binary_to_list(Secret)).
```

Reply:
trim trailing or?

Reply:
I think it's safe to trim both ends.
it's always tricky to have whitespace or CRLF included in secrets

Reply:
Ok. 
My assumption was secret file would be prepared by GitHub and used in github action which should be in good shape.

Reply:
there could be accidents even if it's script generated. e.g.
`echo "secret" > secret-file.`
you get a `\n` in the end.